ci/fft_benchmark_parser.py

```diff
@@ -0,0 +1,203 @@
+"""
+benchmark_parser
+----------------
+
+Parse criterion benchmark.
```

Comment:
Maybe add more context like:
```suggestion
Parse criterion benchmark for tfhe-fft.
```

Reply:
yep